modular_skyrat/master_files/code/modules/mob/living/simple_animal/friendly/dogs.dm

```diff
@@ -44,23 +44,38 @@
 	icon_state = "borgi"
 	icon_living = "borgi"
 	icon_dead = "borgi_dead"
-	maxHealth = 60
-	health = 60
+	unique_pet = TRUE
+	can_be_held = FALSE
+	maxHealth = 150
+	health = 150
 	var/emagged = 0
+	turns_per_move = 10
+	stop_automated_movement = 0
 	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
 	minbodytemp = 0
 	loot = list(/obj/effect/decal/cleanable/oil/slippery)
 	butcher_results = list(/obj/item/clothing/head/corgi/en = 1, /obj/item/clothing/suit/corgisuit/en = 1)
 	deathmessage = "beeps, its mechanical parts hissing before the chassis collapses in a loud thud."
 	animal_species = /mob/living/simple_animal/pet/dog/corgi/borgi
 	nofur = TRUE
+	light_system = MOVABLE_LIGHT_DIRECTIONAL
+	light_color = COLOR_RED
```

Comment:
Why did you give it light totally undocumented, and what does that look like in-game?